src/components/organisms/NewProfileModal/NewProfileModalBody/NewProfileModalBody.scss

```diff
@@ -0,0 +1,2 @@
+.NewProfileModalBody {
+}
```

Comment:
Any reason for this file to exist?

Reply:
nope:) removed in https://github.com/sparkletown/sparkle/pull/2291/commits/0607c2d88914a835fe9998d71b7070b2a765cf3e